policies/branch-policy.md

```diff
@@ -0,0 +1,100 @@
+Branch Policy
+=============
+
+The openssl repository contains the following maintained branches:
+
+### The default development branch
+
+- Any type (bug fix, feature, refactoring, ...) of pull requests is allowed.
+- The development of the next minor or major release happens there.
+  API/ABI breaking changes are allowed on this branch only if the next
+  release will be a major release.
```

Comment:
Does this mean we cut releases from the development branch despite having future major and minor branches?

Reply:
Yes, and after the (final or perhaps beta?) release is cut a future minor or a future major (depending on the next release type) will become the default development branch. That's why the name of the branch never changes and is `openssl-x.y`.